src/components/GlobalPreferences/Notifications/NotificationsPreferences.js

```diff
@@ -0,0 +1,170 @@
+import React, { useCallback } from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconMail,
+  IconTrash,
+  Info,
+  Link,
+  Switch,
+  useTheme,
+  Split,
+  Tag,
+  textStyle,
+} from '@aragon/ui'
+
+import emailNotifcationIllustration from '../../../../src/assets/emailNotifications.svg'
+
+export default function NotificationsPreferences({
+  email,
+  notificationsDisabled,
+  onSwitchNotificationsStatus,
+  onLockSettings,
+  onDeleteEmail,
+  onUpdateEmail,
+}) {
+  const theme = useTheme()
+
+  return (
+    <React.Fragment>
+      <Split
+        primary={
+          <EmailNotificationBox
+            notificationsDisabled={notificationsDisabled}
+            onSwitchNotificationsStatus={onSwitchNotificationsStatus}
+          />
+        }
+        secondary={
+          <React.Fragment>
+            <Box heading="Email account">
+              <Tag
+                uppercase={false}
+                background={theme.infoSurface}
+                size="normal"
+                color={theme.content}
+                css={`
+                  ${textStyle('body2')};
+                  border-radius: 3px;
+                `}
+              >
+                {email}
+              </Tag>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+                wide
+                mode="strong"
+                onClick={onLockSettings}
+              >
+                Lock settings
+              </Button>
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                  justify-content: left;
```

Comment:
Shouldn't this be centered? Same for `Delete your email` Button

Reply:
no, if you center that  they don't look like on the designs

Reply:
Aren't they centered ? https://www.figma.com/file/IdVA6iMn0cWLV5imylwEUP/%E2%9A%96%EF%B8%8F-Court-Dashboard-Master?node-id=4980%3A1815